lib/benchee/conversion.ex

```diff
@@ -51,6 +58,11 @@ defmodule Benchee.Conversion do
       |> Enum.flat_map(fn scenario -> Map.to_list(scenario.run_time_data.statistics) end)
       |> Enum.group_by(fn {stat_name, _} -> stat_name end, fn {_, value} -> value end)
 
+    reductions_measurements =
+      scenarios
+      |> Enum.flat_map(fn scenario -> Map.to_list(scenario.reductions_data.statistics) end)
+      |> Enum.group_by(fn {stat_name, _} -> stat_name end, fn {_, value} -> value end)
```

Comment:
I feel like these could be able to be extracted/reused as I think only the data key changes :thinking: